pkg/rancher-desktop/layouts/dialog.vue

```diff
@@ -68,8 +68,6 @@ export default Vue.extend({
   min-width: 24rem;
   padding: 1.25rem;
   margin: 0 auto;
-  display: flex;
-  flex-flow: column;
```

Comment:
It would be good to have more detail in the commit message about what this is fixing (that is, why this works).